presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -890,4 +890,79 @@ CTE Materialization Properties
 
     The name of the catalog to be used for Common Table Expressions (CTE) and which provides custom partitioning for Common Table Expression (CTE) materialization.
     This setting specifies which catalog should be used for CTE materialization and for determining how to partition the materialization of CTEs in queries.
-    This can also be specified on a per-query basis using the ``cte_partitioning_provider_catalog`` session property.
\ No newline at end of file
+    This can also be specified on a per-query basis using the ``cte_partitioning_provider_catalog`` session property.
+
+
+Logging Properties
+------------------
+
+``log.path``
+^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:** ``var/log/server.log``
+
+    The path to the log file used by Presto. The path is relative to the data
+    directory, configured by the launcher script.
+
+``log.max-history``
+^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``30``
+
+    The `log.max-history` property controls the number of archived log periods that the application retains.
```

Comment:
Let me try and verify once that the same logic(as http-server) also applies to this config. If yes, then we can make the same change here as well that I have mentioned for http-server